bytecode/instr.py

```diff
@@ -269,15 +297,19 @@ def lineno(self):
     def lineno(self, lineno):
         self._set(self._name, self._arg, lineno)
 
-    @property
-    def stack_effect(self):
+    def stack_effect(self, jump=None):
```

Comment:
I do agree this is necessary, but I do not like to break backward compatibility and I tend to prefer verbs for method. But I cannot find a nice one (`eval_stack_effect` could work). WDYT ? If we change the method name we could have a deprecation warning (FutureWarning for visibility).

Reply:
Well you asked what I think so...  Shortly after Stuart Feldman invented "make" somebody pointed out that the tab syntax was horrible.  He said that he agreed, but he didn't want to change because 12 people were already using it.

More seriously:  if we're fans of properties then we could add stack_effect_with_jump and stack_effect_without_jump.  But I dislike that:  the work done in stack_effect is beyond what I think a property ought to do.

We could have a new method with a different name, as you suggested.  And also deprecate stack_effect.  And then once stack_effect is gone we put it back as a method. 

I'm not a fan of "eval_stack_effect" in particular, then name sounds like I might pass it evaluates a stack effect object.  I'd need to go read the docs to understand the difference between the two.

Let me think on this for a few hours, maybe I'll come up with something more constructive.

Reply:
BTW, +1 for your comment "I tend to prefer verbs for method".  

How about "get_stack_method()" ?  On ConcreteInstr.  I like "get" better than "eval", but the irony is that "get" screams "make this a property" :-)

Reply:
I think breaking backward compatibility is not a problem while the version < 1.0.